vignettes/advanced.Rmd

```diff
@@ -7,21 +7,67 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-In this vignette, we discuss how to use the sliding functionality in the 
-`epiprocess` package with computations that have advanced output structures.
-
-In general, the functions `epi_slide()` and `epix_slide()` do what they can to
-ensure the result of a slide operation is *size stable*, meaning, it will return 
-something whose length is the same as the number of appearances of reference 
-time values for the slide computation in the given data frame/table (this 
-defaults to all time values, but can be some given subset when `ref_time_values` 
-is specified).
-
-The output of a slide computation should either be an atomic value/vector, or a 
-data frame. This data frame can have multiple columns, multiple rows, or both. 
-Below we demonstrate some advanced use cases of sliding with these output 
-structures. We focus on `epi_slide()` for the most part, though the behavior we
-demonstrate also carries over to `epix_slide()`.
+In this vignette, we discuss how to use the sliding functionality in the
+`epiprocess` package with less common grouping schemes or with computations that
+have advanced output structures.
+The output of a slide computation should either be an atomic value/vector, or a
+data frame. This data frame can have multiple columns, multiple rows, or both.
+
+During basic usage (e.g., when all optional arguments are set to their defaults):
+
+* `epi_slide(edf, <computation>, .....)`:
```

Comment:
praise: The presentation of `slide` function behavior here is great! It's super useful to be able to compare how particular aspects of data handling differs between `epi_slide` and `epix_slide`. And in the "more complex rules" section, the tags ("clarification", etc) help with readability/skimmabilty.

More generally, looks like you took a very thorough look at documentation and comments :clap: